data/pages.yml

```diff
@@ -10,8 +10,8 @@
   pages:
     - title: "Installing Ember"
       url: "index"
-    - title: "Your First Ember App"
-      url: "first-app"
```

Comment:
Why is this deleted in this commit? I think this was deleted in another PR, too.